docs/MSPX.md

```diff
@@ -19,33 +22,92 @@ mLRS provides the MspX technology, which is designed to improve the over-the-air
 - Set the baudrate to 115200 or higher.
+- INAV 8 and later will also support a baudrate of 230400 but no change in performance or stability was noticed.
   
 <img src="images/MSPX_ports.png" width="720px">
 
-- In the Receiver tab, select the Receiver Type in the Receiver Mode panel to MSP, and save settings.
+- In the Receiver tab, change the Receiver Mode to 'MSP' and the RSSI Source to 'MSP' (INAV 8 and later) then save settings.
   
-<img src="images/MSPX_receivermode.png" width="720px">
+<img src="images/MSPX_receiver.png" width="720px">
 
-- When connecting INAV 8.0 Configurator through mLRS it is highly recommended to enable the Wireless Mode switch before connection, for better link reliability (do not use Wireless Mode with versions earlier than 8.0, such as INAV 7.1).
+- When connecting INAV 8 Configurator through mLRS for flight monitoring, it is recommended to enable the Wireless Mode switch before connection, for better link reliability (do not use Wireless Mode with versions less than 8).
 
 <img src="images/MSPX_wirelessmode.png" width="360px">
 
 If your radio is connected, you should now be able to see the channel values update when you move the radio sticks. No further settings are needed and telemetry will work for EdgeTX/OpenTX radios after scanning for sensors.
+
+## Connection Types with different INAV Versions
+
+Depending on the chosen connection and serial mode, the functionality will slightly differ, especially in INAV versions before 8. This is a quick overview, what options are available and what functions to expect. 
+
+### INAV 8
+
+Serial Mode:
+- MspX
+  - provides telemetry sensors to your radio
+  - allows RC Control over MSP (rc override)
+  - Full MSP GCS Functionality with INAV Configurator, MWP and other applications
+  - comprehensive OSD and Blackbox Link Statistics with dedicated MSP messages provided by mLRS
+  - limited to 50Hz RC Rate (Only relevant for 2.4GHz FLRC Mode with 111Hz Packet rate, Will be limited to 37Hz RC rate)
+- Transparent
+  - See INAV 7 section
+  - Not recommended
+
+RC Out Mode (Can be combined with Serial if the Rx has 2 UART)
+- CRSF
+  - allows RC Control
+  - RC Rate up to 111Hz in 2.4GHz FLRC mode
+
+ ***Recommendation***
+ 
+Since INAV 8 in combination with mLRS can provide all features over MspX, it is recommended to only use a single UART MspX connection for most vehicles. The only exception is the 2.4GHz FLRC mode that can provide 111Hz packet and RC rate. 
+Due to a performance limitation in INAV, this mode had to be limited to 37Hz RC Rate over MSP. Use a separated CRSF UART instead, to achieve full 111Hz RC Update Rates in this mode. All other features will work as normal. 
+
+### INAV 7
+
+Serial Mode: 
+- MspX
+  - provides telemetry sensors to your radio
+  - allows RC Control over MSP (rc override)
+  - limited to 50Hz RC Rate (Only relevant for 2.4GHz FLRC Mode with 111Hz Packet rate, Will be limited to 37Hz RC rate)
```

Comment:
Does it make sense here to align the last 3 bullets with the INAV 8 section?  The first two follow the same order but the last three are moved around.

Reply:
yes makes sense for easier comparison.